src/compiler/symb.bas

```diff
@@ -1977,6 +1987,124 @@ function symbCheckConstAssign _
 	function = TRUE
 end function
 
+function symbCheckConstAssignFuncPtr _
+	( _
+		byval ldtype as FB_DATATYPE, _
+		byval rdtype as FB_DATATYPE, _
+		byval lsubtype as FBSYMBOL ptr, _
+		byval rsubtype as FBSYMBOL ptr, _
+		byval mode as FB_PARAMMODE = 0, _
+		byref matches as integer = 0, _
+		byref errmsg as FB_ERRMSG = 0, _
+		byref wrnmsg as FB_WARNINGMSG = 0 _
+	) as integer
+
+	'' return value:
+	'' TRUE = assignment is comatible
+	'' FALSE = assignment is not compatible
+
+	function = FALSE
+
+	errmsg = FB_ERRMSG_OK '' FB_ERRMSG
+	wrnmsg = 0            '' FB_WARNINGMSG
+
+	assert( typeGetDtOnly( ldType ) = FB_DATATYPE_FUNCTION )
+	assert( typeGetDtOnly( rdType ) = FB_DATATYPE_FUNCTION )
+	assert( symbIsProc( lsubtype ) )
+	assert( symbIsProc( rsubtype ) )
+
+	'' not a function pointer?
+	if( ( typeGetDtOnly( ldType ) <> FB_DATATYPE_FUNCTION ) or ( typeGetDtOnly( rdType ) <> FB_DATATYPE_FUNCTION ) ) then
```

Comment:
(given the assert()s above, this may aswell be removed)

Reply:
ditto as above